LFS171x/sawtooth-material/sawtooth-tuna/client/webpack.config.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path')
```

Comment:
Provide license information for this file.
`// SPDX-License-Identifier: Apache-2.0`